README.md

```diff
@@ -126,7 +133,7 @@ nextflow run phac-nml/ncov-dehoster -profile conda --nanopore --nanostripper --f
 
 You can also just generate dehosted fast5 files with nanostripper with no guppy environment specified. Guppy is proprietary software of ONT Technologies so you must create your own environment for it
 
-Minimum computational specs required for the Nanopore dehosting pipeline are 16+ cpu and 68+G memory
+Minimum computational specs required for the Nanopore dehosting pipeline are 16+ cpu and 68+G memory. GPU highly recommended to run or it will take >12 hours.
```

Comment:
Takes <10 mins with GPU, right?

Reply:
1 - 4 hours typically depending on the amount of data and how many gpus are used. I've updated readme with this info